mera/mera/mera_pymer4.py

```diff
@@ -50,8 +54,10 @@ def run_mera(
         Contains the random effect for each event (rows) and IM (columns)
     site_res_df: dataframe
         Contains the random effect for each site (rows) and IM (columns)
-    me_res_df: dataframe
-        Contains the leftover residuals for each record (rows) and IM (columns)
+        Note: Only returned if compute_site_term is True
+    rem_res_df: dataframe
+        Contains the leftover residuals for
+        each record (rows) and IM (columns)
     bias_std_df: dataframe
         Contains bias, between-event sigma (tau), between-site sigma (phi_S2S),
```

Comment:
I don't have a big picture what context this code will be used, but an added comment will be nice stating bias_std_df has no phi_S2S column if compute_site_term is false